azure-pipelines.yml

```diff
@@ -10,18 +11,14 @@ pr:
     exclude:
     - docs/*
     - THIRD-PARTY-NOTICES.TXT
-    - LICENSE.TXT
+    - License.txt
 
 variables:
   - name: _TeamName
     value: DotNetCore
-  - name: _DotNetArtifactsCategory
-    value: .NETCore
+  - name: _SignType
```

Comment:
This should not be set in public.

Reply:
Why would setting it matter when _SignType is only used by the internal builds?

Reply:
Not a strong stance. I just don't like setting stuff that could get accidentally used in a refactor.